frontend/src/components/historyTable/HistoryTable.jsx

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line
+import React from 'react';
+import Heading from './Heading';
+import Body from './Body';
+import NotFound from '../NotFound';
+import { useStateValue } from '../../contexts/state';
+import { Table } from '../table/Table';
+
+const HistoryTable = () => {
+    const [
+        {
+            historyDataState: { max_build_num },
+        },
+    ] = useStateValue();
+
+    if (max_build_num > 0) {
+        return (
+            <Table table-id="history-table">
+                <Heading />
+                <tbody id="history-table-body">
```

Comment:
The <tbody> has been added everywhere, why not just add it to the Body.jsx ?

Reply:
I actually haven't added anything, only renamed the file, but this is nice improvement 👍